File: data_structures/arraylist/java/arraylist.java
import java.util.*;
public class arraylist {

   public static void main(String args[]) {
      // create an array list
      ArrayList al = new ArrayList();
      System.out.println("Initial size of al: " + al.size());

      // add elements to the array list
      al.add("C");
      al.add("A");
      al.add("E");
      al.add("B");
      al.add("D");
      al.add("F");
      al.add(1, "A2");
      System.out.println("Size of al after additions: " + al.size());

      // display the array list
      System.out.println("Contents of al: " + al);

      // Remove elements from the array list
      al.remove("F");
      al.remove(2);
      System.out.println("Size of al after deletions: " + al.size());
      System.out.println("Contents of al: " + al);
   }
}

File: data_structures/arraylist/kotlin/arraylist.kt
import java.util.ArrayList

fun createArrayList() {

    // Create arrayList
    println("Creating arrayList")
    val arrayList: ArrayList<String> = ArrayList()
    println("Initial size of arrayList: " + arrayList.size)

    // Add arrayList elements
    println("Adding arrayList elements")
    arrayList.add("cheddar")
    arrayList.add("brie")
    arrayList.add("gouda")
    arrayList.add("feta")
    arrayList.add("mozzarella")
    arrayList.add(2, "asiago")
    println("Updated size of arrayList: " + arrayList.size)
    println("Contents of arraylist: $arrayList")

    // Remove elements from the array list
    println("Removing feta")
    arrayList.remove("feta")
    println("Contents of arraylist: $arrayList")
    println("Removing item at index 1")
    arrayList.removeAt(1)
    println("Contents of arraylist: $arrayList")
}
